lib/org/openpsa/invoices/handler/invoice/action.php

```diff
@@ -135,50 +137,70 @@ public function _handler_create_pdf()
         }
     }
 
-    public function _handler_create_pdf_reminder()
+    public function _handler_create_payment_warning()
     {
         $pdf_helper = new org_openpsa_invoices_invoice_pdf($this->invoice);
         try {
             $pdf_helper->render_and_attach('reminder');
-            return $this->reply(true, $this->_l10n->get('reminder pdf created'));
+            return $this->reply(true, $this->_l10n->get('payment warning pdf created'));
         } catch (midcom_error $e) {
-            return $this->reply(false, $this->_l10n->get('reminder pdf creation failed') . ': ' . $e->getMessage());
+            return $this->reply(false, $this->_l10n->get('payment warning pdf creation failed') . ': ' . $e->getMessage());
         }
-    }    
-
-    private function load_send_mail_controller() : controller
+    }
+    
+    private function get_email_type_config() : array
     {
-        $schemadb = schemadb::from_path($this->_config->get('schemadb_send_mail'));
-        $dm = new datamanager($schemadb);
-        
+        $config = [
+            'subject_param' => 'last_' . $this->mail_type . '_subject',
+            'message_param' => 'last_' . $this->mail_type . '_message',
+            'subject_default' => $this->mail_type . '_mail_title_default',
+            'message_default' => $this->mail_type . '_mail_body_default',
```

Comment:
these two lines don't need to be in the config array any more, they're only used in this function now